apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -1141,6 +1141,24 @@ const MultiScrollParent = () => {
   );
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29690&commitId=0e4ee3661d38e66626380001fea6c90690d874c1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Disabled option.default.chromium.png|346| Changed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|413| Changed|
|Combobox Converged.Open.hover.chromium.png|413| Changed|
|Combobox Converged.Option with long content.active option.chromium.png|3915| Changed|
|Combobox Converged.Open.default.chromium.png|417| Changed|
|Combobox Converged.With selection.default.chromium.png|429| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|346| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|417| Changed|
|Combobox Converged.Option with long content.hover.chromium.png|3915| Changed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|413| Changed|
|Combobox Converged.Disabled option.hover.chromium.png|346| Changed|
|Combobox Converged.Open with grouped options.hover.chromium.png|1074| Changed|
|Combobox Converged.With selection.hover.chromium.png|423| Changed|
|Combobox Converged.Open.active option.chromium.png|413| Changed|
|Combobox Converged.Option with long content.default.chromium.png|4091| Changed|
|Combobox Converged.Open with grouped options.active option.chromium.png|1074| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|1079| Changed|
|Combobox Converged.With multiselect selection.default.chromium.png|575| Changed|
|Combobox Converged.With selection.active option.chromium.png|423| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Disabled option.hover.chromium.png|12| Changed|
|Dropdown Converged.Open with grouped options.hover.chromium.png|12| Changed|
|Dropdown Converged.Open with grouped options.default.chromium.png|12| Changed|
|Dropdown Converged.Disabled option.default.chromium.png|12| Changed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|12| Changed|
|Dropdown Converged.Disabled option.active option.chromium.png|12| Changed|
|Dropdown Converged.Open with grouped options.active option.chromium.png|12| Changed|
|Dropdown Converged.Open with inlinePopup.default.chromium.png|12| Changed|
|Dropdown Converged.Open.default.chromium.png|12| Changed|
|Dropdown Converged.Open.active option.chromium.png|12| Changed|
|Dropdown Converged.Open.hover.chromium.png|12| Changed|
|Dropdown Converged.Open with inlinePopup.hover.chromium.png|12| Changed|
|Dropdown Converged.Option with long content.active option.chromium.png|12| Changed|
|Dropdown Converged.Option with long content.hover.chromium.png|12| Changed|
|Dropdown Converged.With selection.active option.chromium.png|12| Changed|
|Dropdown Converged.With selection.hover.chromium.png|12| Changed|
|Dropdown Converged.With selection.default.chromium.png|12| Changed|
|Dropdown Converged.Option with long content.default.chromium.png|12| Changed|
|Dropdown Converged.With multiselect selection.default.chromium.png|12| Changed|

</details>


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.Match target size.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Combobox/Dropdown never applied `box-sizing: border-box` so the width was actually never exactly correct since it did not account for padding.

I can try to create a workaround to avoid visual diffs, but in this case the new result might be more correct WDYT @behowell  @smhigley?